lib/views/answer/single_choice/single_choice_view_model.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:kayla_flutter_ic/views/answer/single_choice/single_choice_option_ui_model.dart';
+import 'package:kayla_flutter_ic/views/answer/single_choice/single_choice_state.dart';
+
+class SingleChoiceViewModel extends StateNotifier<SingleChoiceState> {
```

Comment:
Should we create a ViewModel or State for a type of question? If we do this way, I think we have to create multiple view models / states, which are boilerplate files but bring no benefit 🤔 
Can we only have a view for a type of question, and submit the answer to `question view model` instead?

Reply:
From my perspective, when the user selects an option, that option will be highlighted. 

This means when the user selects an option, the state will be changed, and the view will be rebuilt.

Reply:
Please take a look at [this draft PR](https://github.com/nmint8m/flutter-survey-app/pull/95), I plan to use that approach.

Reply:
@manh-t 

I agree now. I was overthinking about managing the state. 

I simplified the implementation to your suggestion. Just removed the ViewModel and State of the `SingleChoiceView`. Please take a look.